pkg/apis/k8s.cni.cncf.io/v1/types.go

```diff
@@ -162,6 +164,23 @@ type NetworkSelectionElement struct {
 	CNIArgs *map[string]interface{} `json:"cni-args,omitempty"`
 	// GatewayRequest contains default route IP address for the pod
 	GatewayRequest []net.IP `json:"default-route,omitempty"`
+	// IPAMClaimReference container the IPAMClaim name where the IPs for this
+	// attachment will be located.
+	IPAMClaimReference string `json:"ipam-claim-reference,omitempty"`
```

Comment:
I think we need this to be a NamespacedName it could happend that the nad is a one namespace but pod/IPAMClaim at different one.

Also not sure if we may also need passing the UID in the future.

Reply:
That breaks Kubernetes garbage collection, thus, we shouldn't support it.

Why would we want the UID ?

Reply:
Ahh wait this is the network selection element this is not the NAD, ok disregard.